daemon/src/maker.rs

```diff
@@ -80,13 +85,19 @@ async fn main() -> Result<()> {
         ElectrumBlockchain::from(client),
     )
     .unwrap();
-    wallet.sync(NoopProgress, None).unwrap(); // TODO: Use LogProgress once we have logging.
 
     let oracle = schnorrsig::KeyPair::new(SECP256K1, &mut rand::thread_rng()); // TODO: Fetch oracle public key from oracle.
 
     let (cfd_feed_sender, cfd_feed_receiver) = watch::channel::<Vec<Cfd>>(vec![]);
     let (order_feed_sender, order_feed_receiver) = watch::channel::<Option<Order>>(None);
-    let (_balance_feed_sender, balance_feed_receiver) = watch::channel::<Amount>(Amount::ZERO);
+    let (wallet_feed_sender, wallet_feed_receiver) =
+        watch::channel::<WalletInfo>(WalletInfo::Syncing);
+
+    // initial wallet sync (blocking, so we do it before kicking things off)
+    wallet.sync(NoopProgress, None).unwrap(); // TODO: Use LogProgress once we have logging.
+
+    // TODO: We should get addresses on demand from a POST endpoint
+    let _address = wallet.get_address(AddressIndex::New).unwrap().address;
```

Comment:
Should we do [`LastUnused`](https://docs.rs/bdk/0.11.0/bdk/wallet/enum.AddressIndex.html#variant.LastUnused) instead?

Reply:
This line is actually a relic that should be removed 😅